lib/exw3.ex

```diff
@@ -225,6 +225,15 @@ defmodule ExW3 do
     end
   end
 
+  @spec get_filter_logs(binary()) :: any()
+  @doc "Gets event changes (logs) by filter. Unlike ExW3.Contract.get_filter_logs it does not return the data in a formatted way"
+  def get_filter_logs(filter_id) do
```

Comment:
@maxwolff should this be called `get_logs` and use the RPC function call `eth_getLogs`? I don't see an RPC function called `eth_getFilterLogs`.